packages/app/src/app/components/SandboxList/index.js

```diff
@@ -56,8 +56,8 @@ export default ({ sandboxes, isCurrentUser, onDelete }: Props) => (
               <Link to={sandboxUrl(s)}>{s.title || s.id}</Link>
               <PrivacyStatus privacy={s.privacy} asIcon />
             </td>
-            <td>{moment(s.insertedAt).format('ll')}</td>
-            <td>{moment(s.updatedAt).format('ll')}</td>
+            <td>{format(s.insertedAt, 'DD/MM/YYYY')}</td>
```

Comment:
This is a different format than `ll` (which is `5 Jan 2017` for example), is there a way to get that with `date-fns` as well? It would clear up a lot of confusion between US/EU.

Reply:
You need this one `MMM DD, YYYY`